pcsx2/R5900.h

```diff
@@ -231,21 +231,21 @@ struct tlbs
 
 #define _PC_       cpuRegs.pc       // The next PC to be executed - only used in this header and R3000A.h
 
-#define _Funct_  ((cpuRegs.code      ) & 0x3F)  // The funct part of the instruction register
-#define _Rd_     ((cpuRegs.code >> 11) & 0x1F)  // The rd part of the instruction register
-#define _Rt_     ((cpuRegs.code >> 16) & 0x1F)  // The rt part of the instruction register
-#define _Rs_     ((cpuRegs.code >> 21) & 0x1F)  // The rs part of the instruction register
-#define _Sa_     ((cpuRegs.code >>  6) & 0x1F)  // The sa part of the instruction register
-#define _Im_     ((u16)cpuRegs.code) // The immediate part of the instruction register
-#define _Target_ (cpuRegs.code & 0x03ffffff)    // The target part of the instruction register
+#define _Funct_			((cpuRegs.code      ) & 0x3F)  // The funct part of the instruction register
+#define _Rd_			((cpuRegs.code >> 11) & 0x1F)  // The rd part of the instruction register
```

Comment:
Unnecessary formatting changes

Reply:
I think this is just classic "tabs look awful".  In visual studio it appears to have done this to properly line things up 
![image](https://user-images.githubusercontent.com/13153231/102161963-4f17c700-3e56-11eb-821b-66f169f4cefe.png)
If i run clang on it, it removes this style of formatting which I've seen push back against before so I think we'll want to leave this.

Reply:
If you mean pushback from me, I like the alignment but would prefer if it was made with spaces (like the previous ones were)

Tabs are for indentation, not alignment

Reply:
Ah i see now, I wouldn't have expected visual studio to change the entire alignment to tabs when running a simple find/replace so I just assumed it added similar characters to correctly align.  I'll fix that.  Sorry I'm not used to looking at whitespace this closely in diffs, I'm used to working exclusively with spaces.